pom.xml

```diff
@@ -276,6 +276,12 @@
                     <version>2.8.1</version>
                     <configuration>
                         <argLine>${argLine} -Djava.awt.headless=true -noverify</argLine>
+                        <includes>
+                            <include>**/*.java</include>
```

Comment:
Shouldn't be the case. The default inclusion list only includes the first three lines I put in my comment.